_posts/2020-01-30-new-age-of-bastion.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: "New age of Bastion"
+author: Mahmut Bulut
+---
+
+**Disclaimer:** This post is not only an announcement of new features of Bastion, which comes with 0.3.4; it also declares our future perspective for the project.
+
+So we had a long way since our project had started in June 2019. Right now, we are evolving from a runtime that enables fault-recovery for your server-side applications to an agnostic runtime, which has its executor but also works with other executors. Since the beginning we can interoperate with [async-std](https://async.rs) without the hassle.
+
+We have started with an idea of runtime, which can recover from failures in the application domain, no matter what happens. Async/await landed, and we got stronger and well known in the community. Then Bastion deployed to AWS lambdas, did, and still doing its job in data processing pipelines. Benchmarks of sparse queries outperformed plenty of sparse queries and aggregations over NYC taxi data in various inmem benchmarks.
+
+Where futures-executor performs:
+```
+test count_aggregation      ... bench: 108,916,527 ns/iter (+/- 11,909,528)
+```
+Bastion performed:
+```
+test count_aggregation      ... bench: 87,096,485 ns/iter (+/- 3,874,812)
+```
+All have started with an idea of replicating what Erlang does its runtime. Using the runtime for the data processing comes right after it. Writing a generic runtime for fault-tolerant middleware, distributed systems, and data processing pipelines was the main goal of Bastion.
```

Comment:
All have started with an idea of replicating what Erlang does its runtime.
Everything started / it all started with ?

Using the runtime for the data processing comes right after it.
I m not native, but i couldnt understand this sentence :/